app/models.py

```diff
@@ -31,6 +32,46 @@ def password(self, password):
 	def verify_password(self, password):
 		return check_password_hash(self.password_hash, password)
 
+	def save(self):
+		"""add user instance to session and save to databas"""
+		db.session.add(self)
+		db.session.commit()
+
+	def generate_token(self, user_id):
+		"""a method to generate the access token"""
+		try:
+			# set up a payload
+			payload={
+				'exp': datetime.utcnow() + timedelta(minutes=10),
+				'iat': datetime.utcnow(),
+				'sub': user_id
+			}
+			# create the byte string token using the payload and the SECRET key
+			jwt_string = jwt.encode(
+				payload,
+				current_app.config.get('SECRET_KEY'),
+				algorithm='HS256'
+			)
+			return jwt_string
+
+		except Exception as e:
```

Comment:
use pylint to show you best practices for naming conventions, like `e` should raise a concern